### DISCUSSION TITLE
Content rating update job has updated the rating in neo4j as well as redis, elastic search as individual content level but the same was not updated in cassandra content hierarchy

### DISCUSSION THREAD

**Discussion by kamal-singhal** [Sunbird-Ed/Community#663] [Bugs]
Hi Team,

Content rating update job has updated the rating in neo4j as well as redis, elastic search as individual content level but the same not updated in Cassandra content hierarchy.
Can you please help let us there are any job that will synch the data neo4j to Cassandra.
can you share any document or KT video where we can understand the complete workflow of content rating update in database and how to come on the UI level.

**Comment by swayangjit** [comment]
@pallakartheekreddy can you please check whom we can assign this issue ?

**Reply by pallakartheekreddy** [reply]
@kamal-singhal Content rating update job will not update in Cassandra (hierarchy data) this is expected behaviour only because hierarchy update is a costly operation, and it will impact performance.
Currently, We don't have any tools or jobs synch the data neo4j to Cassandra.
